.github/workflows/ci-workflow.yml

```diff
@@ -17,247 +17,173 @@ name: ci-workflow
 
 on: [push, pull_request]
 
+env:
+  DEFAULT_MAVEN_OPTS: >-
+    -Xmx2g
+    -XX:ReservedCodeCacheSize=1g
+    -XX:+UseG1GC
+    -Dorg.slf4j.simpleLogger.defaultLogLevel=WARN
+    -Dhttp.keepAlive=false
+    -Dmaven.wagon.http.pool=false
+  MAVEN_COMMAND: >-
+    mvn package verify
+    -V
+    -D dir-only
+    -D disableXmlReport
+    -nsu
+    -ntp
+    -ff
+  MAVEN_PROFILES: >-
+    -P contrib-check
+    -P include-grpc
+  MAVEN_PROJECTS: >-
+    -pl -nifi-assembly
+    -pl -nifi-toolkit/nifi-toolkit-assembly
+    -pl -nifi-registry/nifi-registry-assembly
+    -pl -minifi/minifi-assembly
```

Comment:
This line is very important, I didn't realize we hadn't been building the minifi-assembly before, I missed it with the big integration PR. Great catch!